rust/src/nasl/syntax/statement.rs

```diff
@@ -536,4 +544,153 @@ mod position {
 
         assert_eq!(tests, expected.len());
     }
+
+    #[track_caller]
+    fn test_statement(name: &str, code: &str) {
+        insta::with_settings!({ prepend_module_to_snapshot => false }, {
+            let mut parser = parse(code);
+            let stmt = parser.next().unwrap().unwrap();
+            assert_snapshot!(format!("statement_{name}"), stmt);
```

Comment:
I changed the format of the snapshot to use the Display implementation instead. But now I have the feeling, that valuable information like the actual statement kind is missing, and most of the test just look like the input statement.

Reply:
isn't that good that it almost looks identical? I mean it is a representation of it.

Reply:
The reason why I think it is a good practice is because that is exactly what a customer will see; if it has insufficient information, although we can immediately map it to the test case, there is a high change that it has insufficient information for the end customer as well, considering that they usually don't have the luxury of a well named test case to map the output to.